File: apzkr-pzpi-21-4-nazarko-danylo/Task4-MobileClient/app/src/main/java/ua/nure/danylo/nazarko/pzpi_21_4/washingmachinemanagementmobile/models/WashingMachine.kt
package ua.nure.danylo.nazarko.pzpi_21_4.washingmachinemanagementmobile.models

import kotlinx.serialization.Serializable

@Serializable
data class WashingMachine(
    val id: String,
    val name: String,
    val manufacturer: String,
    val serialNumber: String,
    val description: String? = null
)

File: apzkr-pzpi-21-4-nazarko-danylo/Task1-Server/WashingMachineManagementApi.Domain/Entities/WashingMachine.cs
namespace WashingMachineManagementApi.Domain.Entities;

public class WashingMachine : EntityBase<string>
{
    public string? Name { get; set; }

    public string? Manufacturer { get; set; }

    public string? SerialNumber { get; set; }

    public string? Description { get; set; }

    public DeviceGroup? DeviceGroup { get; set; }

    public string? DeviceGroupId { get; set; }
}
